Shared/Supporting Files/Views/SampleDetailView.swift

```diff
@@ -16,14 +16,20 @@ import SwiftUI
 
 struct SampleDetailView: View {
     /// The sample to display in the view.
-    private let sample: Sample
+    let sample: Sample
```

Comment:
Xcode was throwing 'using variable before it is set' when `isFullScreen` was directly in the init and the workarounds were convoluted. Let me know if the access level of the Sample is okay to be exposed or if there is a better way to handle the init.